site/zenodo_rdm/metrics/utils.py

```diff
@@ -24,7 +24,10 @@ def calculate_metrics(metric_id, cache=True):
     result = deepcopy(current_app.config["METRICS_DATA"][metric_id])
 
     for metric in result:
-        metric["value"] = metric["value"]()
+        try:
+            metric["value"] = metric["value"]()
+        except:
+            current_app.logger.exception(f"Metric evaluation failed {metric['name']}")
```

Comment:
```suggestion
            current_app.logger.exception(f"Metric evaluation failed: %s", metric['name'])
```

...or...

```suggestion
            current_app.logger.exception("Metric evaluation failed", extra={"metric": metric['name']})
```

minor: this helps [Sentry to group](https://docs.sentry.io/platforms/python/usage/sdk-fingerprinting/) errors better (otherwise the exception message changes everytime). I know that we have only one metric type, but it's good to have it right in case someone is looking on how to log an exception and copy-pastes this :)